src/gluonts/ev/ev_tutorial.md

```diff
@@ -0,0 +1,162 @@
+# New evaluation approach
+
+This notebook explains how to use the `ev` module which is a new alternative to the `evaluation` module in GluonTS.
+
+## [Usage](usage_example.py)
+
+Before diving into how things work, let's look at how to use the new evaluation approach.
```

Comment:
I think a brief example is great, but then let's also do this: Have a simple example demonstrating the principle.